app/forms/candidate_interface/continuous_applications/course_selection_wizard.rb

```diff
@@ -0,0 +1,29 @@
+module CandidateInterface
+  module ContinuousApplications
+    class CourseSelectionWizard < DfE::Wizard
+      attr_accessor :current_application
+
+      steps do
+        [
```

Comment:
Why is this an `Array` and not a single `Hash`?

Reply:
I added because I think we might consider pass options to each step. I think even if don't make sense to be an Array, we will have cases of other wizards to add options to the steps via key value.